pkg/wireguard/wireguard.go

```diff
@@ -71,6 +71,11 @@ func NewDevice(address netaddr.IPPrefix, privateKey wgtypes.Key, listenPort uint
 		if err == nil {
 			logger.Sugar().Info("using native Wireguard device")
 
+			err = setupIPToInterface(dev)
```

Comment:
this should be done on any exit from this function, so it should include exit below with user-space implementation

Reply:
Ooops - sorry, partial commit. Should be fixed now.